Modules/selectmodule.c

```diff
@@ -282,7 +282,7 @@ select_select(PyObject *self, PyObject *args)
                 n = 0;
                 break;
             }
-            _PyTime_AsTimeval_noraise(timeout, &tv, _PyTime_ROUND_CEILING);
+            _PyTime_AsTimeval_noraise(timeout, &tv, _PyTime_ROUND_UP);
```

Comment:
Since `timeout` >= 0, there is no difference between `_PyTime_ROUND_CEILING` and `_PyTime_ROUND_UP` here.

Reply:
Done in 5c96cd6